app-manifest.yml

```diff
@@ -27,11 +27,14 @@ oauth_config:
       - chat:write.public
       - groups:history
       - reactions:read
+      - im:read
 settings:
   event_subscriptions:
     request_url: https://change-this-later.example.com/slack/events
     bot_events:
       - reaction_added
+      - message.channels
```

Comment:
Adding these makes sense for the changes here but I'm still reluctant to add these in the default settings. Having a different App Manifest for auto-translate feature users can be an option.